_api-reference/cat/cat-indices.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: default
-title: CAT indices operation
+title: CAT indexes operation
 parent: CAT API
 nav_order: 25
 has_children: false
 redirect_from:
 - /opensearch/rest-api/cat/cat-indices/
 ---
 
-# CAT indices
+# CAT indexes
```

Comment:
@kolchfa-aws: Could I get your take here? Are we ok changing the title and name of the API without changing the URL? The endpoint says `indices`.

Reply:
@Naarcha-AWS I agree with you. We should keep it `indices` because that's the name of the endpoint.

Reply:
Reversed the changes!